src/librustc/session/config.rs

```diff
@@ -855,6 +866,7 @@ pub fn rustc_optgroups() -> Vec<RustcOptGroup> {
                  "NAME=PATH"),
         opt::opt("", "sysroot", "Override the system root", "PATH"),
         opt::multi("Z", "", "Set internal debugging options", "FLAG"),
+        opt::opt_u("", "output", "How errors and other mesasges are produced", "tty|json"),
```

Comment:
I suspect we should have `--output` as the long form of `-o`, so maybe this could be `--error-format` or something? Also, the "tty" format might be better as "human", since e.g. json can go to a tty and the human-readable version can go to a file.

("Mesasges" is spelt wrong.)